src/codegen/program.lisp

```diff
@@ -36,10 +36,18 @@
           :if (and (typep thing '(cons (member cl:defstruct)))
                    (sub-struct-p thing))
             :collect thing :into defsubstructs
-          ;; (DEFSTRUCT FOO ...
+          ;; (DEFSTRUCT FOO () ...
```

Comment:
nit: `defstruct foo ()` is invalid syntax; it was correct before because it doesn't have an `:include` clause